pkg/sql/parser/builtins.go

```diff
@@ -1442,18 +1442,22 @@ var Builtins = map[string][]Builtin{
 	// and the session's database search path.
 	"current_schemas": {
 		Builtin{
-			Types:        ArgTypes{{"unused", TypeBool}},
```

Comment:
Hah, I think you guys are just commenting based on the `-` diff here. I've just renamed the argument because it's no longer unused after this commit.